src/main/java/org/helpberkeley/memberdata/ControlBlock.java

```diff
@@ -382,22 +382,32 @@ private void processAuditControl(String variable, String value, long lineNumber)
     }
 
     //
-    // An OpManager data field should look like "userName | phone number"
+    // An OpManager data field should look like "username" or "userName | phone number"
     //
     private void processOpsManager(final String value, long lineNumber) {
 
         String[] fields = value.split("\\" + INTRA_FIELD_SEPARATOR, -42);
 
-        if (fields.length != 2) {
-            throw new MemberDataException("OpsManager value \"" + value
-                    + "\" at line " + lineNumber + " does not match \"username | phone\".\n");
-        }
+//        if (fields.length != 2) {
+//            throw new MemberDataException("OpsManager value \"" + value
+//                    + "\" at line " + lineNumber + " does not match \"username | phone\".\n");
+//        }
```

Comment:
Remove these commented out lines of code.  In general (there are exceptions) it is wrong to commit commented out code.  For example here, this will only serve to confuse some future you or me looking at this code after we have forgotten about this set of changes.  Also, remember that we are using a version control system and it is easy to look at older versions of this file to see what these lines looked like.